tests/unit/src/test/scala/tests/FindTextInFilesSuite.scala

```diff
@@ -0,0 +1,38 @@
+package tests
+
+import scala.meta.internal.metals.Directories
+
+import org.eclipse.lsp4j.Location
+import org.eclipse.lsp4j.Position
+import org.eclipse.lsp4j.Range
+
+class FindTextInFilesSuite extends BaseLspSuite("find-text-in-jar-files") {
+  test("find exact string match in .conf file inside jar") {
+    val expectedUri =
+      s"${workspace.resolve(Directories.dependencies)}/akka-actor_2.12-2.6.16.jar/reference.conf"
+    val expectedLocations: List[Location] = List(
+      new Location(
+        expectedUri,
+        new Range(new Position(95, 2), new Position(95, 20))
+      ),
+      new Location(
+        expectedUri,
+        new Range(new Position(1177, 40), new Position(1177, 58))
+      )
+    )
+    for {
+      _ <- initialize(
+        s"""/metals.json
+           |{
+           |  "a": {
+           |    "scalaVersion": "2.12.4",
+           |    "libraryDependencies": ["com.typesafe.akka::akka-actor-typed:2.6.16"]
+           |  }
+           |}
+        """.stripMargin
+      )
+      locations <- server.findTextInFiles(".conf", "jvm-shutdown-hooks")
+      _ = assertEquals(locations, expectedLocations)
```

Comment:
In metals tests, usually received and expected results are rendered into String representation that is more readable.
Could you do assert in a similar way to how it's done in `WorkspaceSymbolLspSuite`? https://github.com/scalameta/metals/blob/55161a500b77946e4052a91e38479ad42dd285ff/tests/unit/src/test/scala/tests/WorkspaceSymbolLspSuite.scala#L160-L198

Reply:
Not yet done, moved file so this change is not really outdated, will fix this